package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ibm-ldap",
-  "version": "1.0.1",
+  "version": "2.0.1",
```

Comment:
I did it using the command npm version major -m "Reason". Still if you want I can put it back.